### DISCUSSION TITLE
Loading Tiff file throws an exception

### DISCUSSION THREAD

**Discussion by ddyakovsyndy** [SixLabors/ImageSharp#2408] [Q&A] [answered]
Hello,

I am trying to mutate some Tiff files but when I call `await Image.LoadAsync(stream, ctx)`, it throws the exception `System.NotSupportedException: ExtraSamples is only supported with one extra sample for alpha data.`.

Any idea what might be causing this and how it can be fixed?

Thank you in advance for the help.

**Comment by JimBobSquarePants** [comment, marked as answer]
I'd say the exception is telling you exactly what this means. 

>System.NotSupportedException

The image is not supported.

>[ExtraSamples](https://www.awaresystems.be/imaging/tiff/tifftags/extrasamples.html) is only supported with one extra sample for alpha data

Your tiff file contains more than a single entry for this tag which is not supported.

The only way this can be "fixed" is for someone to update the decoder to support this scenario.

**Reply by ddyakovsyndy** [reply]
Thank you for the rapid response!

**Comment by rwetzold** [comment]
I get this error now as well. Would be amazing if that could be supported in the future.

**Reply by JimBobSquarePants** [reply]
@rwetzold  Can you please provide a sample image? I'd like to do some investigation.

**Reply by rwetzold** [reply]
Sorry a bit late but here you go
[Armature_Legs_MetallicSmoothness.zip](https://github.com/user-attachments/files/20941398/Armature_Legs_MetallicSmoothness.zip)

**Comment by JimBobSquarePants** [comment]
As of version v3.1.10 the library will no longer throw an exception and correctly decode images like this.

https://github.com/SixLabors/ImageSharp/releases/tag/v3.1.10

**Reply by rwetzold** [reply]
This sounds great! Is there a chance to backport this to 2.x as we are locked into .netstandard2 with Unity?
